### ISSUE TITLE
ToolBox Controller GUI

### ISSUE THREAD

**Issue by RoeyMaor** [SpartanRefactoring/Main#1388]
<a href="https://github.com/amirsagiv83"><img src="https://avatars2.githubusercontent.com/u/15860047?v=3" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [amirsagiv83](https://github.com/amirsagiv83)**
_Sunday May 14, 2017 at 18:57 GMT_
_Originally opened as https://github.com/TechnionYP5777/Leonidas-FTW/issues/222_

----

We need to implement a brand new ToolBox controller, based on the existing one to add Edit features for the tippers.

We started working on it.

**Comment by RoeyMaor** [comment]
<a href="https://github.com/amirsagiv83"><img src="https://avatars2.githubusercontent.com/u/15860047?v=3" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [amirsagiv83](https://github.com/amirsagiv83)**
_Sunday May 14, 2017 at 19:00 GMT_

----

We reached to a point where we need to implement a special container that will have different types of components in it.

After reading about it, we decided to try and work on 2 different approaches:
1. JList
2. JTable.

Each of us will implement one of the above, and after this is done, we will know how to continue.

**Comment by michalcohen** [comment]
Blocked by #1438

**Comment by amirsagiv83** [comment]
So, I think we are finally done here.
Edit tipper will probably be re-designed, but a new issue will deal with it.
Saving the info will be managed in #1539
